flink-java/src/main/java/org/apache/flink/api/java/DataSet.java

```diff
@@ -521,7 +521,7 @@ public long count() throws Exception {
 		}
 
 		return new ReduceOperator<>(this, new SelectByMinFunction(
-				(TupleTypeInfo) getType(), fields), Utils.getCallLocationName());
```

Comment:
I have already reverted the style change. The only thing is that it was reverted on top of the previous commit. How to totally avoid this change from appearing from my commit history? Am not sure how to do it.